README.md

```diff
@@ -313,18 +313,38 @@ class Messages extends Translations {
 
 #### Using translations
 
-Just append `.tr` to the specified key and it will be translated, using the current value of `Get.locale` and `Get.fallbackLocale`.
+Just append `.tr` to the specified key, and it will be translated, using the current value of `Get.locale` or `Get.fallbackLocale`.
 
 ```dart
 Text('title'.tr);
 ```
 
 ### Locales
 
+Flutter, by default, only supports English language `Locale` (*en_XX*).
```

Comment:
Just to be more precise, could you tell us in what situations this occurs?
When I use the GetMaterialApp locale in PT_br, I have no exceptions

Reply:
Idk why it happens to me.

Reply:
I think what should be changed in the documentation is just the removal of this part:
```dart
  ...
  supportedLocales: <Locale> [Locale ('en', 'UK'), Locale ('en', 'US'), Locale ('de', 'DE')]
  ...
```
Because it is not necessary for GetX's Internationalization feature and it still causes an error that needs to be resolved with flutter_localizations.

supportedLocales is useful for Material components, for example, for the calendar component to show the names of months in the defined language

Reply:
@eduardoflorence 
I used GetX package in my app. now try to implement Calendar with locale but it's give me some error.
Error:- "No MaterialLocalizations Found. DatePickerDialog widget require MaterialLocalization to be provided by a Localization widget ancestor".

Please support me how i can fix that error?